packages/semantic-model/test/unit/api-negative.test.ts

```diff
@@ -17,12 +17,12 @@ describe("The ui5-language-assistant semantic model package API negative tests",
     ): () => UI5SemanticModel {
       const generateCall = (): UI5SemanticModel => {
         return generate({
-          version: "1.71.49",
+          version: "1.71.61",
```

Comment:
Wouldn't it be better to have a constant for the version here as well?

Reply:
Created a constant